packages/jsii-calc/lib/compliance.ts

```diff
@@ -1335,3 +1335,34 @@ export class Constructors {
         return new PrivateClass();
     }
 }
+
+// internal can be used to represent members that can only be accessed from the current module
+export class StripInternal {
+    public youSeeMe = 'hello';
+
+    /**
+     * This is an internal thing
+     * @internal
```

Comment:
Are we doing this as a replacement for names starting with a `_`? Additional way to achieve the same behavior?